### DISCUSSION TITLE
Version for Spring 25

### DISCUSSION THREAD

**Discussion by FabrizioMoggio** [camaraproject/CallForwardingSignal#115] [General]
Considering that this release is an outcome from a development phase of v0.2.0, I propose to list it **0.3.0: Stable** in the Spring25 release Tracker.

**Comment by FabrizioMoggio** [comment]
Considering: https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/14557919/API+versioning

Specifically the following text: "public (x.y.z) API versions for usage in commercial contexts. Public API versions only have API version number x.y.z (semver 2.0), no extension. The public API can have one of two maturity states: 

initial (x=0) - indicating that the API is still not fully stable

stable (x>0) - indicating that the API has reached a certain level of maturity"

**We should produce v1.0.0**

**Comment by FabrizioMoggio** [comment]
Stable version only if we CK Hutchison gets the certification of v0.2.0 from GSMA before 20th of February

**Comment by FabrizioMoggio** [comment]
On M3 we will produce the release candidate: v1.0.0-rc.1
https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/14559630/API+Release+Process#API-releases---overview

If the API will be certified by GSMA. 

Otherwise we will produce a v0.3.0-rc.1

**Comment by FabrizioMoggio** [comment]
@StefanoFalsetto-CKHIOD news? :-) Do you think w can proceed with a "stable version" (v1.0.0-rc.1, if the certification process of v0.2.0 is completed) or we need to still go for a "initial version" (v0.3.0-rc.1). We need to provide the M3 PR (with the version) for the 17th of January.

**Reply by StefanoFalsetto-CKHIOD** [reply]
As you can see the GSMA put our affiliates operators on the map, but I am asking them why they put TBC and no other details. They still didn't started testing. I don't know whether this is currently sufficient for a 1.0 version.

<img width="1467" alt="Screenshot 2025-01-09 alle 09 30 27" src="https://github.com/user-attachments/assets/11e1e303-2764-4277-90e9-15c53b2be209" />

**Reply by FabrizioMoggio** [reply]
First of all this is a great result, I would like to compliment with you.

About the release number, please keep us updated and , if you can, please check with GSMA.

**Comment by FabrizioMoggio** [comment]
Maybe I can create a second PRs (for v0.3.0), so we have both the options ready and then, on the 16th of January, we can decide which one to keep and which one to abort.

**Comment by FabrizioMoggio** [comment]
I got a suggestion in today's CAMARA Hall Hands call. the certification of the API is not mandatory for a RC. It is mandatory for the final stable release. So @hdamker suggested me to move forward with the review of v1.0.0-rc.1 (M3). Before M4 we will very if the certification is positively completed and we will decide to have the API released as Stable (v1.0.0) or Initial (v0.3.0).

A note about the ongoing certification must be added in the RCL.

**Comment by FabrizioMoggio** [comment]
Considering https://github.com/camaraproject/CallForwardingSignal/discussions/146#discussioncomment-11829208 let's go the other way around. We can move forward with v0.3.0. If something changes before M4 we can still update the API to v1.0.0.
